src/views/forms/AccountManagementForms/DepositForm.tsx

```diff
@@ -374,6 +375,15 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
     }
 
     if (routeErrors) {
+      track(
+        AnalyticsEvents.RouteError({
+          transferType: TransferType.deposit.name,
+          errorMessage: routeErrorMessage?.toString(),
+          amount: debouncedAmount,
+          chainId: chainIdStr?.toString(),
```

Comment:
supernit, this is already a string, so I guess `chainIdStr || undefined` might be more accurate

Reply:
oh that's my bad - i was using `toString` because i thought it was a kotlin string type conversion issue but it's a `Nullable` type conversion issue 😮  makes sense! will fix all subsequent calls as well